wpeverest/user-registration
# UR-233 Feature - hCaptcha

### All Submissions:






### Changes proposed in this Pull Request:

Introducing hCaptcha, Which helps to make more secure bots as Google ReCaptcha.

### How to test the changes in this Pull Request:

1. Goto Integration and select recaptcha type hCaptcha and enter site key and secret Key and then save settings.
2. Enable recaptcha in Login OR Registration Form
3. Visit Login Page and you will see the hCaptcha.
4. Test validation error messages and Ajax Login too.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Introducing hCaptcha
